lib/kafka/client.rb

```diff
@@ -436,15 +436,16 @@ def initialize_cluster
       )
     end
 
-    def build_ssl_context(ca_cert, client_cert, client_cert_key)
+    def build_ssl_context(ca_cert, client_cert, client_cert_key, client_cert_key_passphrase)
       return nil unless ca_cert || client_cert || client_cert_key
 
       ssl_context = OpenSSL::SSL::SSLContext.new
 
       if client_cert && client_cert_key
         ssl_context.set_params(
           cert: OpenSSL::X509::Certificate.new(client_cert),
-          key: OpenSSL::PKey.read(client_cert_key)
+          key: OpenSSL::PKey.read(client_cert_key, client_cert_key_passphrase)
```

Comment:
If `ssl_client_cert_key_passphrase` is not pass then default  value `nil` is saved. Also openssl tend to use passphrase only when key is encrypted. If it is not encrypted then passphrase provided is not used.
https://github.com/ruby/openssl/blob/7b319d95ed46a9ffb1d9962d5d4de8bff94cb28c/ext/openssl/ossl_pkey_rsa.c#L210